IceUpdater Warnings in debug

/Users/slauae/projects/moose/framework/src/base/FEProblem.C:1167:101: warning: unused parameter 'auto_parsed' [-Wunused-parameter]
FEProblem::addFunction(std::string type, const std::string & name, InputParameters parameters, bool auto_parsed )
n file included from /Users/slauae/projects/moose/framework/src/outputs/ICEUpdater.C:18:
In file included from /Users/slauae/projects/moose/framework/contrib/ice_updater/include/Updater.h:44:
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/INetworkingTool.h:56:39: warning: unused parameter 'url' [-Wunused-parameter]
  virtual std::string get(std::string url, std::string username,
                                      ^
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/INetworkingTool.h:56:56: warning: unused parameter 'username' [-Wunused-parameter]
  virtual std::string get(std::string url, std::string username,
                                                       ^
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/INetworkingTool.h:57:17: warning: unused parameter 'password' [-Wunused-parameter]
    std::string password)
                ^
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/INetworkingTool.h:73:40: warning: unused parameter 'url' [-Wunused-parameter]
  virtual std::string post(std::string url, std::string value,
                                       ^
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/INetworkingTool.h:73:57: warning: unused parameter 'value' [-Wunused-parameter]
  virtual std::string post(std::string url, std::string value,
                                                        ^
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/INetworkingTool.h:74:15: warning: unused parameter 'username' [-Wunused-parameter]
  std::string username, std::string password)
              ^
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/INetworkingTool.h:74:37: warning: unused parameter 'password' [-Wunused-parameter]
  std::string username, std::string password)
                                    ^
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/INetworkingTool.h:88:50: warning: unused parameter 'ignoreSslPeerVerification' [-Wunused-parameter]
  virtual void setIgnoreSslPeerVerification(bool ignoreSslPeerVerification)
                                                 ^
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/INetworkingTool.h:101:36: warning: unused parameter 'val' [-Wunused-parameter]
  virtual void setNoProxyFlag(bool val)
                                   ^
In file included from /Users/slauae/projects/moose/framework/src/outputs/ICEUpdater.C:18:
/Users/slauae/projects/moose/framework/contrib/ice_updater/include/Updater.h:430:25: warning: unused parameter 'stream' [-Wunused-parameter]
  Updater(std::istream &stream) { do { std::ostringstream _error_oss_; _error_oss_ << "\n\n" << (Moose::_color_console ? "\33[31m" : "") << "\n\n*** ERROR ***\n" << "Updater requires --enable-cxx11 parameter to update_and_build_libmesh.sh." << (Moose::_color_console ? "\33[39m" : "") << "\n\n"; if (Moose::_throw_on_error) throw std::runtime_error(_error_oss_.str()); else { Moose::err << _error_oss_.str() << std::flush; if (libMesh::global_n_processors() == 1) print_trace(); else libMesh::write_traceout(); do { libMesh::MacroFunctions::here("/Users/slauae/projects/moose/framework/contrib/ice_updater/include/Updater.h", 430, "nodate", "notime"); } while(0); MPI_Abort(libMesh::GLOBAL_COMM_WORLD,1); exit(1); } } while (0); }
                        ^
10 warnings generated.

--- a/framework/contrib/ice_updater/include/Updater.h
+++ b/framework/contrib/ice_updater/include/Updater.h
@@ -429,7 +429,7 @@ public:
    * The constructors all throw errors.
    */
   Updater() { mooseError("Updater requires --enable-cxx11 parameter to update_and_build_libmesh.sh."); }
-  Updater(std::istream &stream) { mooseError("Updater requires --enable-cxx11 parameter to update_and_build_libmesh.sh."); }
+  Updater(std::istream &) { mooseError("Updater requires --enable-cxx11 parameter to update_and_build_libmesh.sh."); }
 
   /**
    * The following functions do nothing, and will never be called.

--- a/framework/moose.mk
+++ b/framework/moose.mk
@@ -126,7 +126,7 @@ exodiff: app_INCLUDES := $(exodiff_includes)
 exodiff: $(exodiff_APP)
 
 $(exodiff_APP): $(exodiff_objects)
-	@echo "Linking "$@"..."
+	@echo "Linking Executable "$@"..."
 	@$(libmesh_LIBTOOL) --tag=CXX $(LIBTOOLFLAGS) --mode=link --quiet \
 	  $(libmesh_CXX) $(libmesh_CPPFLAGS) $(libmesh_CXXFLAGS) $(libmesh_INCLUDE) $(exodiff_objects) -o $@ $(libmesh_LIBS) $(libmesh_LDFLAGS) $(EXTERNAL_FLAGS)
 

